libs/vkd3d/state.c

```diff
@@ -4827,7 +4905,8 @@ VkPipeline d3d12_pipeline_state_create_pipeline_variant(struct d3d12_pipeline_st
         library_create_info.flags = library_flags;
 
         pipeline_desc.pNext = &library_create_info;
-        pipeline_desc.flags |= VK_PIPELINE_CREATE_RETAIN_LINK_TIME_OPTIMIZATION_INFO_BIT_EXT;
+        pipeline_desc.flags |= VK_PIPELINE_CREATE_LIBRARY_BIT_KHR |
```

Comment:
This should be moved to the first commit to avoid confusion.

Reply:
done